### PR TITLE
Setup Styling & Images

### PR DESCRIPTION
**Pull request by numan-7**
- Configured Tailwind Config File & Imported Design Images
- Just added small small subheadings under Frontend Roadmap on where to find the changes just for future reference.

### CODE REVIEW

File: codewit/client/index.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Client</title>
+    <title>Codewit.us</title>
```
Comment (kbuffardi) [inline codewit/client/index.html:5]: Let's stick with branding that's consistent with the (all lowercase) logo: `codewit.us`

File: ROADMAP.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ This document provides a "roadmap" to help developers navigate the organization
 
 The `codewit/client` directory contains the react app that drives the front-end.
 
+#### Tailwind Styling Configurations
+
+- All styling configurations for Tailwind are located under `codewit/client/src/tailwind.config.js`
+
+#### Website Icon/Logos
+
+- The site icon, or favicon, which appears in the browser tab, is located under the `codewit/client/public/` directory alongside the logo and hexicon. The icon is currently initialized in `codewit/client/index.html` along with the website title.
```
Comment (kbuffardi) [inline ROADMAP.md:15]: These descriptions are fine, although let's cut out the last sentence. Pretty soon we'll develop a header bar and the favicon will probably "live" there.

**Review by kbuffardi — CHANGES_REQUESTED** [review]
Just a couple minor revisions.

**Comment by numan-7** [comment]
> Just a couple minor revisions.

Resolved both of them.

**Review by kbuffardi — APPROVED** [review]

**Pull request merged**